src/RolloutScorer/RolloutScorerAzureFunction.Tests/RolloutScorerAzureFunctionTests.cs

```diff
@@ -0,0 +1,41 @@
+using FluentAssertions;
+using Microsoft.Azure.KeyVault;
+using Microsoft.Azure.Services.AppAuthentication;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Timers;
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+
+namespace RolloutScorerAzureFunction.Tests
+{
+    public class RolloutScorerAzureFunctionTests
+    {
+        private RolloutScorerFunction _rolloutScorerFunction;
+
+        [SetUp]
+        public void Setup()
+        {
+            Mock<IKeyVaultClient> keyVaultClientMock = new Mock<IKeyVaultClient>();            
+
+            var services = new ServiceCollection()
```

Comment:
Really this should be disposed... it... probably doesn't matter?  You could also so services.GetRequiredService<RolloutScorerFunction> instead of the activator thingy, since it doesn't have any weird parameters. :-)